templates/data/mongodb/managed.html

```diff
@@ -0,0 +1,466 @@
+                            loading="lazy",
+                            attrs={"class": "p-logo-section__logo"}) | safe
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image(url="https://assets.ubuntu.com/v1/9faa9b91-aws-logo.png",
+                            alt="AWS",
+                            width="125",
+                            height="256",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-logo-section__logo"}) | safe
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image(url="https://assets.ubuntu.com/v1/497cabc1-google-cloud-logo.png",
+                            alt="Google Cloud",
+                            width="397",
+                            height="256",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-logo-section__logo"}) | safe
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image(url="https://assets.ubuntu.com/v1/f7a0c5ff-kubernetes-logomark-logo.png",
+                            alt="Kubernetes",
+                            width="105",
+                            height="384",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-logo-section__logo"}) | safe
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image(url="https://assets.ubuntu.com/v1/165fa040-openstack-logo.png",
+                            alt="OpenStack",
+                            width="426",
+                            height="256",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-logo-section__logo"}) | safe
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image(url="https://assets.ubuntu.com/v1/a89f2843-microcloud.png",
+                            alt="Microcloud",
+                            width="394",
+                            height="312",
+                            hi_def=True,
+                            loading="lazy",
+                            attrs={"class": "p-logo-section__logo"}) | safe
+              }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width p-section--shallow">
+      <hr />
+      <div class="row">
+        <h2>
+          Why choose Canonical's
+          Managed
+          <br class="u-hide--medium u-hide--small" />
+          MongoDB
+        </h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-9 col-medium-6 col-start-large-4">
```

Comment:
The list section should be wrapped in a shallow section
<img width="534" alt="Screenshot 2024-08-22 at 1 48 43 PM" src="https://github.com/user-attachments/assets/fd2f9547-e972-456c-8ac0-6b1cb99d074b">